services/productService.js

```diff
@@ -0,0 +1,10 @@
+const { getProductsById } = require("../models/productDao");
+
+const productService = async (productId) => {
```

Comment:
service 레어이의 함수명을 다음과 같이 작성하면 service 레이어 자체와 헷갈릴 가능성이 있을 것 같습니다.

좀 더 명확한 함수명을 작성해주는 것이 좋을 것 같습니다!
```suggestion
const getProductById = async (productId) => {
```